plugins/console/index.js

```diff
@@ -0,0 +1,102 @@
+var q = require('q');
```

Comment:
I think you should use protractor.promise instead of q.  The webdriver code has its own promise implementation and you probably want to use the same infrastructure.